### PR TITLE
[WIP] post for next week advocating people try out the beta.

### PR DESCRIPTION
**Pull request by pnkfelix**

### CODE REVIEW

File: posts/2021-05-27-incremental-progress.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+recommend seeing the summary [issue #84970][].
+
+[issue #84970]: https://github.com/rust-lang/rust/issues/84970
+
+I am proud to report that we have now backported the relevant fixes to the 1.53
+beta channel. Therefore, starting with the 1.53.0 release, we will resume
+providing incremental-compilation support *without* requiring use of the
+`RUSTC_FORCE_INCREMENTAL` environment variable.
+
+## 1.53-beta: Not Just For βreakfast Anymore
```
Comment (bjorn3) [inline posts/2021-05-27-incremental-progress.md:48]:
```suggestion
## 1.53-beta: Not Just For Breakfast Anymore
```
This makes it more readable with screenreaders.

Comment (pnkfelix) [reply posts/2021-05-27-incremental-progress.md:48]: Oh, that's a good point. I should probably change the "βetta" down below to "Betta" too.

File: posts/2021-05-27-incremental-progress.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+but who *does* care about Rust's success, here is my final pitch to you: we, the
+Rust compiler team, want as many people as possible to use the 1.53 beta release
+now, so that we find out about any previously unidentified "fingerprint"
+problems well before the 1.53.0 stable release comes out on June 17th. So, we
+recommend that you, the Rust enthusiasts, try switching to the 1.53-beta release
+of Rust. (And, of course, file issues if you see any bugs!)
+
+[law of the excluded middle]: https://en.wikipedia.org/wiki/Law_of_excluded_middle
+
+## βetta Than Evvah
```
Comment (pnkfelix) [inline posts/2021-05-27-incremental-progress.md:74]:
```suggestion
## Betta Than Evvah
```

**Review by Manishearth — APPROVED** [review]

**Comment by pnkfelix** [comment]
Note: we are still working out the timing for when we will post this, if at all. At the very least, the beta needs some more backports before we would pro-actively advertise everyone use it in this manner.

I'm going to put this into draft mode until we have resolved the above question.

**Comment by Mark-Simulacrum** [comment]
Doesn't look like we'll be posting this.

**Pull request closed without merging**